h/api/schemas.py

```diff
@@ -211,7 +211,7 @@ def validate(self, data):
             new_appstruct['shared'] = _shared(appstruct.pop('permissions'),
                                               new_appstruct['userid'])
         else:
-            new_appstruct['shared'] = True
+            new_appstruct['shared'] = False
```

Comment:
Yeah, when updating the default behaviour is to not changed the shared value from whatever is currently in the DB.